File: official/docs/csharp/v4/webhooks/delete.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;
using EasyPost;
using Newtonsoft.Json;

namespace EasyPostExamples
{
    public class Examples
    {
        public static async Task Main()
        {
            var client = new EasyPost.Client("EASYPOST_API_KEY");

            EasyPost.Models.API.Webhook webhook = await client.Webhook.Retrieve("hook_...");

            bool success = await webhook.Delete();

            Console.WriteLine(success.ToString());
        }
    }
}

File: official/docs/java/v7/webhooks/delete.java
package webhooks;

import com.easypost.exception.EasyPostException;
import com.easypost.service.EasyPostClient;

public class Delete {
    public static void main(String[] args) throws EasyPostException {
        EasyPostClient client = new EasyPostClient("EASYPOST_API_KEY");

        client.webhook.delete("hook_...");
    }
}
